server/adaptors/integrations/__test__/builder.test.ts

```diff
@@ -307,6 +314,13 @@ describe('IntegrationInstanceBuilder', () => {
         tags: undefined,
         creationDate: expect.any(String),
         assets: refs,
+        references: [
+          {
+            id: undefined,
+            name: undefined,
+            type: 'data-source',
+          },
+        ],
```

Comment:
Ideally, we shouldn't add a reference if Instance was build without MDS. Should remove this altogether without MDS.